Makefile

```diff
@@ -152,16 +152,6 @@ godoc: ## Display documentation for the package
 	@printf "\n"
 	godoc --http=localhost:6060 --play
 
-cluster.opendistro.build:
-	docker-compose --project-directory .ci/opendistro build;
-
-cluster.opendistro.start:
-	docker-compose --project-directory .ci/opendistro up -d ;
-	sleep 20;
-
-cluster.opendistro.stop:
-	docker-compose --project-directory .ci/opendistro down ;
-
 cluster.opensearch.build:
```

Comment:
nit : May be we can call it as cluster.build?

Reply:
Done.